.bazelrc

```diff
@@ -0,0 +1,9 @@
+build --cxxopt=-std=c++17 --copt=-march=core-avx2
```

Comment:
You are affecting files at the root of google-research. We probably can't do this.

Reply:
Yes, of course, I have removed all these things. And nobody else is going to use bazel